src/microsoft-extensions-ai-evaluation/api/INSTRUCTIONS.md

```diff
@@ -60,7 +60,7 @@ The following setup steps are required to set up the LLM connection / endpoint t
 
 1. **Select your LLM provider:** The examples are authored to run against Azure Open AI by default and have been tested
    against GPT-4o. You can easily switch to use Azure AI Inference, Ollama, or Open AI by changing one line of code
-   within [`TestSetup.GetChatConfiguration()`](./evaluation/Setup/TestSetup.cs#L25).
+   within [`TestSetup.GetChatConfiguration()`](https://github.com/dotnet/ai-samples/blob/9d28db621f110d51914645e0fe9c733942e9107f/src/microsoft-extensions-ai-evaluation/api/evaluation/Setup/TestSetup.cs#L32).
```

Comment:
I am not sure we should do this. If the code is changed, I'd rather not point to an older version of the code (since thre code may have changed significantly).

Instead, it may be better to add a comment in the code file sayting that the line is referenced in the md file and that the md file should be updated if the code / line numbers end up changing.

Happy to handle this in one of my uipcoming PRs.

Reply:
I have included a commit for this in #108 (cbed367)